src/vs/workbench/contrib/debug/browser/repl.ts

```diff
@@ -428,6 +455,7 @@ export class Repl extends ViewPane implements IHistoryNavigationWidget {
 		this.replInputContainer.style.height = `${replInputHeight}px`;
 
 		this.replInput.layout({ width: width - 30, height: replInputHeight });
+		this.filterState.layout = new dom.Dimension(width, height);
```

Comment:
I do not understand why we update teh filterStat layout when in practice this does not seem to do anything ofr me.
Also filterState and filter should idealy not be attributes on the repl but just local variables passed to the tree.

Reply:
I think it is required for this state change, but I not sure how it is implemented for markers filter.

<img width="585" alt="Screenshot 2020-08-16 at 12 50 00" src="https://user-images.githubusercontent.com/14974179/90333584-0abb0e00-dfbf-11ea-83ab-258b35868b15.png">
<img width="572" alt="Screenshot 2020-08-16 at 12 49 55" src="https://user-images.githubusercontent.com/14974179/90333586-0c84d180-dfbf-11ea-91f5-674421b40abc.png">